File: demo-vk-rest-scraping.java/src/main/java/demo/model/Chair.java
package demo.model;

public class Chair {

    private final int id;
    private final String title;

    private Chair(int id, String title) {
        this.id = id;
        this.title = title;
    }

    public Chair(ConstructorMediator cm) {
        this(cm.getId(), cm.getTitle());
    }

    public int getId() {
        return id;
    }

    public String getTitle() {
        return title;
    }

    @Override
    public String toString() {
        return "Chair{" +
                "id='" + id + '\'' +
                ", title='" + title + '\'' +
                '}';
    }
}

File: demo-vk-rest-scraping.kotlin/src/main/kotlin/demo/model/Chair.kt
package demo.model

class Chair private constructor(val id: Int, val title: String) {

    constructor(cm: ConstructorMediator) : this(cm.id, cm.title) {
    }

    override fun toString(): String {
        return "Chair{" +
                "id='" + id + '\'' +
                ", title='" + title + '\'' +
                '}'
    }
}
